export const createUser = async(data: {}) => {
    try{

    }catch(err: any) {
        console.log(err);
        
    }
}